beakerx/beakerx/install.py

```diff
@@ -67,15 +72,10 @@ def _install_kernels():
 
     for kernel in _all_kernels():
         kernel_classpath = _classpath_for(kernel)
-        classpath = os.pathsep.join([base_classpath, kernel_classpath])
-        # workaround #5864
-        if sys.platform == 'win32':
-            classpath = classpath.replace('\\', '/')
-        # #5859: replace with string.Template, though this requires the
-        # developer install to change too, so not doing right now.
+        classpath = json.dumps(os.pathsep.join([base_classpath, kernel_classpath]))
         template = pkg_resources.resource_string(
             'beakerx', os.path.join('static', 'kernel', kernel, 'kernel.json'))
-        contents = template.decode().replace('__PATH__', classpath)
+        contents = Template(template.decode()).substitute(PATH="\"" + classpath + "\"")
```

Comment:
this doesn't look right, json.dumps returns a value with quotes already.

Reply:
done